.github/workflows/issues2md.yml

```
@@ -0,0 +1,27 @@
+# Action to backup issues using gh2md, contributed by
+# https://github.com/0ut0fcontrol.
+# See https://github.com/mattduck/gh2md/issues/11.
+name: Issues2Markdown
+on:
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
+        fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+    - name: Backup github issues to a markdown file.
+      run: |
+        pip3 install --user --upgrade setuptools
+        pip3 install --user gh2md
+        $HOME/.local/bin/gh2md $GITHUB_REPOSITORY issues.md --idempotent
+        git add issues.md
+      env:
+        GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Archive issues export
+      uses: actions/upload-artifact@v2
+      with:
+        name: issues export
+        path: issues.md
```

This workflow works in my forked repo. You can see the run and output here: https://github.com/Coffee2Bits/AaC/actions/runs/1856072006
```suggestion
name: Export Issues to Markdown
on:
  workflow_dispatch:

jobs:
  build:
    name: Export Issues to Markdown
    runs-on: ubuntu-latest
    steps:
    - name: Backup Github Issues to Markdown
      run: |
        pip3 install --user --upgrade setuptools
        pip3 install --user gh2md
        $HOME/.local/bin/gh2md $GITHUB_REPOSITORY issues.md --idempotent
      env:
        GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    - name: Archive Issues Report
      uses: actions/upload-artifact@v2
      with:
        name: issues_export
        path: issues.md
```